CliClient/locales/ko.po

```diff
@@ -1477,30 +1475,27 @@ msgid "Solarised Dark"
 msgstr "Solarized Dark"
 
 msgid "Nord"
-msgstr ""
+msgstr "노드"
 
-#, fuzzy
 msgid "Show note counts"
-msgstr "트레이 아이콘 표시"
+msgstr "노트 수 보이기\""
 
-#, fuzzy
 msgid "Editor"
 msgstr "편집"
 
-#, fuzzy
 msgid "Viewer"
-msgstr "보기"
+msgstr "뷰어"
 
 msgid "Split View"
-msgstr ""
+msgstr "뷰 나누기"
 
-#, fuzzy, javascript-format
+#, javascript-format
 msgid "%s / %s / %s"
 msgstr "%s = %s (%s)"
```

Comment:
This seems strange to me. I'm not quite sure how `%s / %s / %s` would become `%s = %s (%s)` in Korean. Are you sure this is ok?

Reply:
When you resolve a conversation, please add a comment, so that we know what's going on.

Reply:
You resolved the conversation again w/o anything. Ok, let me put this more easily. If you can't do what I ask, then we have to do it this way: You don't resolve a conversation. I do.